airflow/contrib/sensors/gcs_sensor.py

```diff
@@ -67,6 +69,10 @@ def ts_function(context):
     behaviour is check for the object being updated after execution_date +
     schedule_interval.
     """
+    schedule_interval = context['dag'].schedule_interval
+    if isinstance(schedule_interval, str):
+        iterator = croniter(schedule_interval, context['execution_date'])
+        return iterator.get_next(datetime)
```

Comment:
if this current date time (not initialised here it seems), use datetime.now(pytz.utc)

(yes pytz needs to be added)